en/documentation/index.md

```diff
@@ -79,71 +142,114 @@ syntax-highlighting, file browsing) or an integrated development
 environment with advanced features (e.g. code completion, refactoring,
 testing support).
 
-Here is a list of popular tools used by Rubyists:
-
-* Linux and cross-platform tools:
-  * [Aptana Studio][19]
-  * [Atom][atom]
-  * [Emacs][20] with [Ruby mode][21] and [Rsense][22]
-  * [Eric][eric]
-  * [Geany][23]
-  * [gedit][24]
-  * [Kate][kate]
-  * [KDevelop][kdevelop]
-  * [NetBeans][36]
-  * [RubyMine][27]
-  * [SciTe][28]
-  * [Sublime Text][37]
-  * [Vim][25] with [vim-ruby][26] plugin and [Rsense][22]
-  * [Visual Studio Code][vscode]
-
-* On Windows:
-  * [Notepad++][29]
+Here is a list of popular editors used by Rubyists, broken up by
+learning curve:
 
-* On macOS:
-  * [TextMate][32]
-  * [BBEdit][33]
-  * [Dash][39] (documentation browser)
+* Days
+  * [Sublime Text][37] (paid)
+  * [Visual Studio Code][vscode]
+  * [Zed][zed]
+* Months
+  * [RubyMine][27] (paid)
+* "Years" (as in, you'll spend years still learning things about it)
+  * [Emacs][20] with [Ruby mode][21] or [Enhanced Ruby mode][enh-ruby-mode]
+  * [Vim][25] with [vim-ruby][26] plugin
+  * [NeoVim][neovim]
+
+All of these editors support the Language Server Protocol (LSP),
+either by default or through their LSP plugins. Shopify's
+[ruby-lsp][ruby-lsp] is one of the most popular language servers for
+Ruby and [supports all of the above editors][shopify_lsp].
 
 ### Further Reading
 
+[Official FAQ](/en/documentation/faq/)
+: The official frequently asked questions.
+
 [Ruby-doc.org][34] maintains a comprehensive list of English
```

Comment:
I think this and link 34 can be removed because it's been listed in the `Reference Documentation` section and it's not really a "further reading" anyway.